### DISCUSSION TITLE
How to disabled all mouse event?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#4792] [Q&A]
How to disabled all mouse event?

**Comment by willmcgugan** [comment]
`mouse=False`

https://textual.textualize.io/api/app/#textual.app.App.run(mouse)

**Reply by Yurii-huang** [reply]
![image](https://github.com/user-attachments/assets/72f5d7f9-f4dd-4b1c-9275-2bc6dc6e45be)

version: 0.43.2

**Reply by Yurii-huang** [reply]
run() got an unexpected keyword argument 'mouse'

**Reply by willmcgugan** [reply]
You are using an old version of Textual. Please upgrade to the latest version.
